### PR TITLE
EPMRPP-39353: incorrect time values when filtering launches

### PR DESCRIPTION
**Pull request by Andrew-Omelchenko**

### CODE REVIEW

File: app/src/components/filterEntities/entityItemStartTime.jsx [hunk 1]
```diff
@@ -39,12 +33,9 @@ const presets = [
     value: {
       start: moment()
         .startOf('day')
-        .subtract(1, 'weeks')
+        .subtract(6, 'days')
```
Comment (DiscoElevator) [inline app/src/components/filterEntities/entityItemStartTime.jsx]: Why 6 days?

File: app/src/components/filterEntities/entityItemStartTime.jsx [hunk 2]
```diff
@@ -53,18 +44,15 @@ const presets = [
     value: {
       start: moment()
         .startOf('day')
-        .subtract(1, 'months')
+        .subtract(29, 'days')
```
Comment (DiscoElevator) [inline app/src/components/filterEntities/entityItemStartTime.jsx]: Why 29 days? Months have different day count.

File: app/src/components/filterEntities/entityItemStartTime.jsx (hunk 1 above)
Comment (Andrew-Omelchenko) [reply app/src/components/filterEntities/entityItemStartTime.jsx]: Expected:
"filter.btw.start_time":"-1440;362;+0300" ('1440' is number of minutes of the previous day,'362' is number of minutes till the current moment of filtering and +0300 is the user's timezone)
Launches with start time in that range are shown. It was for Last 2 days. So for Last 7 days we should send "-1440*6;number of minutes till the current moment of filtering;+0300"

File: app/src/components/filterEntities/entityItemStartTime.jsx (hunk 2 above)
Comment (Andrew-Omelchenko) [reply app/src/components/filterEntities/entityItemStartTime.jsx]: In dropdown user sees not 1 month, but `30 days`.

**Comment by DiscoElevator** [comment]
Waiting for clarification with QA and BA

**Comment by Andrew-Omelchenko** [comment]
Fixed. Exactly as in v4.

**Pull request merged**